server/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -44,27 +47,59 @@ public AdminProgramBlocksController(
       ProgramService programService,
       QuestionService questionService,
       ProgramBlockEditView editView,
+      ProgramBlockEditView readOnlyView,
```

Comment:
This comment should stay open until the view is replaced with a read only version (blocked unit the other pull request is submitted)